docs/overview.md

```diff
@@ -13,22 +16,49 @@ As a versatile solution, ScalarDB supports a range of databases, including:
 
 For details on which databases ScalarDB supports, refer to [Supported Databases](scalardb-supported-databases.md).
 
+## Why ScalarDB?
+
+There are several solutions, such as global transaction managers, data federation engines, and HTAP systems, built towards similar goals, but they are limited in the following perspectives:
+
+- Global transaction managers (e.g., Oracle MicroTx, Atomikos) are designed to run transactions across a limited set of heterogeneous databases (e.g., only XA-compliant databases).
+- Data federation engines (e.g., Denodo, Starburst) are designed to run analytical queries across heterogeneous databases.
+- HTAP systems (e.g., TiDB, SingleStore) run both transactions and analytical queries only on homogeneous databases.
```

Comment:
We should avoid `e.g.,` and use `for example`, `such as`, or `like` instead. (Reference: [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg))

```suggestion
- Global transaction managers (like Oracle MicroTx and Atomikos) are designed to run transactions across a limited set of heterogeneous databases (like only XA-compliant databases).
- Data federation engines (like Denodo and Starburst) are designed to run analytical queries across heterogeneous databases.
- HTAP systems (like TiDB and SingleStore) run both transactions and analytical queries only on homogeneous databases.
```

Reply:
@josh-wong Thanks! 
Do you know why we should avoid it?
It's commonly used in academic papers.